docs/proposals/PROPOSAL_E.md

```diff
@@ -179,6 +230,7 @@ For registries that do not support the `referrers` API, a tag MUST be pushed for
 - `<ref>`: the digest from the `refers` field (limit of 64 characters)
 - `<hash>`: the digest of this artifact (limit of 16 characters)
 - `<type>`: type of artifact for filtering (limit of 5 characters)
+  - The type field is based on a mapping from `artifactType` to a short string for the tag.
```

Comment:
this is probably to go away depending on https://github.com/opencontainers/wg-reference-types/issues/72